sdk/appconfiguration/azure-appconfiguration/samples/sync_token_samples_async.py

```diff
@@ -24,16 +24,18 @@ async def handle_event_grid_notifications(event_grid_events):
 
     all_keys = []
 
-    async with AzureAppConfigurationClient.from_connection_string(CONNECTION_STRING) as client:
```

Comment:
This will cause connection unclose.

We recommend users to use context manager to close client.

Reply:
There is no `__enter__` (or `__aenter__` in async) on the clients. Adding this now.